tests/python_functional/src/syslog_ng_config/renderer.py

```diff
@@ -20,6 +20,129 @@
 # COPYING for details.
 #
 #############################################################################
+from src.syslog_ng_config.statements.filters.filter import Filter
+
+
+def render_version(version):
+    return "@version: {}\n".format(version)
+
+
+def render_includes(includes):
+    include_lines = ['@include "{}"'.format(include) for include in includes]
+    return "\n".join(include_lines)
+
+
+def render_global_options(global_options):
+    globals_options_header = "options {\n"
+    globals_options_footer = "};\n"
+
+    config_snippet = globals_options_header
+    for option_name, option_value in global_options.items():
+        config_snippet += "    {}({});\n".format(option_name, option_value)
+    config_snippet += globals_options_footer
+
+    return config_snippet
+
+
+def render_positional_options(positional_parameters):
+    config_snippet = ""
+    for parameter in positional_parameters:
+        config_snippet += "        {}\n".format(str(parameter))
+    return config_snippet
+
+
+def render_dict(name, options):
+
+    config_snippet = ""
+    config_snippet += "        {}(\n".format(name)
+    config_snippet += render_driver_options(options)
+    config_snippet += "        )\n"
+
+    return config_snippet
+
+
+def render_list(name, options):
+    config_snippet = ""
+
+    for element in options:
+        config_snippet += "        {}({})\n".format(name, element)
+
+    return config_snippet
+
+
+def render_simple(name, value):
```

Comment:
If possible, could we change the function name? Maybe some config related name would be better.

Reply:
Fixed.